### PR TITLE
Handle defaultValue NULL for MySQL driver (with unit test)

### PR DESCRIPTION
**Pull request by axelduch**
I also updated another test (with 9 columns -> with 10 columns) since it was impacted.

### CODE REVIEW

**Comment by axelduch** [comment]
Should fix #254

File: test/driver/mysql_test.js [hunk 1]
```diff
@@ -119,7 +125,7 @@ driver.connect(config, internals, function(err, db) {
           assert.equal(column.getDataType(), 'TINYINT');
           assert.equal(column.isNullable(), true);
           assert.equal(column.getDefaultValue(), 0);
-        }
+        },
```
Comment (axelduch) [inline test/driver/mysql_test.js]: Oops sorry about the extra comma

Comment (axelduch) [inline test/driver/mysql_test.js]: Sorry about the extra comma, it needs deletion

Comment (wzrdtales) [inline test/driver/mysql_test.js]: Then push the change to your branch ;)

I will take a look at this in about two hours, currently still busy.

**Comment by wzrdtales** [comment]
Considered as bugfix. Is also going to be released under v0.9.12 as soon as all travis tests passed.

**Pull request merged**